doc/code_style/mom.h

```diff
@@ -62,7 +62,7 @@ class YourMom : public MassiveEntity {
 	 * Creates your mom.
 	 * @param mass: Initial mass for this mother.
 	 */
-	YourMom(int mass=DEFAULT_MOM_MASS);
+	YourMom(int mass = DEFAULT_MOM_MASS);
```

Comment:
it should be like python does it, with no spaces around the `=` for default values.

Reply:
`SpaceBeforeAssignmentOperators: 'false'`

Reply:
Didn't find any other option than the mentioned one? So I think we would need to agree on one of these:

- either having no space before the assignment operators, but after it
- or having spaces before and after

In my opinion `before and after looks` more "clean" (like in symmetric), what do you think?

Reply:
Assignment operators in general should have spaces before and after, only in the case of default argument values there should be none (like "[Don't use spaces around the = sign when used to indicate a keyword argument, or when used to indicate a default value for an unannotated function parameter](https://www.python.org/dev/peps/pep-0008/)").
If clang-format doesn't support this, maybe we can persuade it to leave default argument values alone?

Reply:
We could turn it off I think, but that would mean as well, that the other option spaces before and after for assignments will not be automatically done anymore. We base the `.clang-format` file on the LLVM style, it could be, that it's done already there - would need to try that. I assume we need to decide if we take the approach that spaces for assignments are made automatically, as they are probably much more in the code. And tell people manually to not use spaces. No one will be forced to use the `.clang-format` any way. 

I see `clang-format` as the minimum standard of coding-style this repository should have with what we agree on that people, that are not coding regularly, can PR to us somehow.